### ISSUE TITLE
cublas error

### ISSUE THREAD

**Issue by re-unknown** [Haoming02/sd-webui-forge-classic#132]
When using cublas, it works fine with batch count 2 and size 2,
but with batch count 2 and size 1, the generated images appear overfitted.
It seems there may be an issue with the processing when the batch size is 1.

<img width="768" height="1408" alt="Image" src="https://github.com/user-attachments/assets/3e5475d8-88b0-4aae-95d9-f6fea91889bb" />

Batch count 1, size 1.
SEED: 1586368888.

<img width="768" height="1408" alt="Image" src="https://github.com/user-attachments/assets/f0fe39cb-2378-42a3-a227-9f00f8ff58c7" />

Batch count 1, size 2.
SEED: 1586368888.

The prompt and seed are the same, but the pose is also different.

**Comment by Haoming02** [comment]
Could you instead try disabling `cublas` and enabling `fp16_accumulation`?

In my experience, the fast `fp16` provides the same speed up as `cublas_ops`, but the former is implemented natively in PyTorch, and thus has less issues.

**Comment by re-unknown** [comment]
Thinking that there would be no conflict, I set --fast-fp16 and cublas at the same time.
Currently, cublas is still set to ON, but since there is no problem as long as it is not generated with size1, I will just report it.
I will also explain that I am using the RTX20 series.

**Comment by Haoming02** [comment]
Having both on is redundant, so you can disable `cublas` just fine.

And both should work on RTX 20s.

**Comment by re-unknown** [comment]
Thank you for your cooperation!
It was a great learning experience!
